keps/sig-apps/961-maxunavailable-for-statefulset/README.md

```diff
@@ -320,11 +607,17 @@ maxUnavailable to a number greater than 1,  but the invariants and the logic wil
 maxUnavailable pods with the same identity and never more than maxUnavailable being deleted.
 
 ###### What specific metrics should inform a rollback?
-TODO when we reach Beta
+
+When feature enabled but rolling update in a unexpected phenomenon like the update pods at a time is not equal to the
+`maxUnavailable` value (we may have some other factors impacting this, but this should apply for most of the time) or
+rolling update in a unexpected order.
```

Comment:
I'm not sure I understand what you mean with this sentence? Can you make it more clear?